lua/entities/magicbutton.lua

```diff
@@ -292,7 +304,15 @@ end
 function ButtonMoneyPrize()
     local min, max = 1000, 300000
 
-    return math.Round(math.pow(math.Rand(0, 1), 4) * (max - min) + min, -3)
+    return string.Comma(math.Round(math.pow(math.Rand(0, 1), 4) * (max - min) + min, -3))
+end
+
+local function MagicOutcomePrize(ply)
+    local amount = ButtonMoneyPrize()
+    if (ply.SS_GivePoints == nil) then return nil end
+    ply:SS_GivePoints(amount)
```

Comment:
So you're trying to give a string value with commas to them?

Reply:
oh fuck that wasn't supposed to go there.